product/src/controllers/ProductController.js

```diff
@@ -1,45 +1,37 @@
-const database = require('../models');
+import { Router } from 'express';
+import { createProduct } from '../usecase/createProductUseCase.js';
+import { listProducts } from'../usecase/listProductsUseCase.js';
+import jwt from 'jsonwebtoken';
+import { createUser } from '../usecase/createUserUseCase.js';
+import { Users } from '../models/users.js';
+import validationToken from '../middlewares/auth.js';
+import bcrypt from 'bcryptjs';
```

Comment:
are you using all these imports?

Reply:
No, unnecessary imports have been deleted.